src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"gui.securitycraft:mrat": "Outil d’accès à distance pour les mines",
+	"gui.securitycraft:srat.notBound": "Non associé!",
+	"gui.securitycraft:srat.sentryLocations": "Sentinelle à #location",
+	"gui.securitycraft:srat.name": "Outil d’accès à distance pour les sentinelles",
+	"gui.securitycraft:srat.modifyAll": "Modifier toutes les sentinelles associées",
+	"gui.securitycraft:password.invalidCode": "Code Invalide!",
+	"gui.securitycraft:password.setup": "Installation",
+	"gui.securitycraft:camera.activateNightVision": "Activer la vision nocturne",
+	"gui.securitycraft:camera.exit": "Exit Caméra",
+	"gui.securitycraft:camera.toggleredstone": "Basculer le signal redstone",
+	"gui.securitycraft:camera.toggleredstoneNote": "(nécessite un module redstone)",
+	"gui.securitycraft:camera.zoom": "Zoomer/dézoomer",
+	"gui.securitycraft:monitor.selectCaméras": "Sélectionner la camera à voir:",
+	"gui.securitycraft:monitor.cameraName": "Nom de Caméra: #",
+	"gui.securitycraft:monitor.cameraInDifferentDim": "Vous pouvez ne pas voir cette caméra maintenant, car cette caméra est dans une autre dimension avec l’ID #.",
+	"gui.securitycraft:briefcase.setupTitle": "Entrer un code à 4 chiffres:",
+	"gui.securitycraft:briefcase.enterPasscode": "Entrez le code:",
+	"gui.securitycraft:universalKeyChanger.enterNewPasscode": "Entrez le nouveau code:",
+	"gui.securitycraft:universalKeyChanger.confirmNewPasscode": "Confimer le nouveau code:",
+	"gui.securitycraft:universalKeyChanger.confirm": "Confirmez",
+	"gui.securitycraft:blockReinforcer.title": "Blindeur de blocs universel",
+	"gui.securitycraft:editModule": "Éditer le Module",
+	"gui.securitycraft:editModule.add": "Ajouter",
+	"gui.securitycraft:editModule.remove": "Supprimer",
+	"gui.securitycraft:editModule.copy": "Copier la Liste",
+	"gui.securitycraft:editModule.paste": "Coller la Liste",
+	"gui.securitycraft:editModule.clear": "Effacer la Liste",
+	"gui.securitycraft:blockPocketManager.activate": "Activer",
+	"gui.securitycraft:blockPocketManager.deactivate": "Désactiver",
+	"gui.securitycraft:blockPocketManager.size": "Taille: %sx%sx%s",
+
+	"messages.securitycraft:bugReport": "Vous pouvez signaler un plantage ou un bug en utilisant ce formulaire GoogleDocs",
+	"messages.securitycraft:securityCaméra.mounted": "Vous êtes maintenant connectés sur une caméra. Utilisez les touches WASD pour déplacer la vue de la caméra et les boutons +/- pour zoomer et dézoomer.",
+	"messages.securitycraft:retinalScanner.hello": "Bonjour #.",
+	"messages.securitycraft:retinalScanner.notOwner": "Je suis désolé, vous ne pouvez pas utiliser ce scanner. Il appartient à #.",
+	"messages.securitycraft:portableRadar.withName": "#p est près de votre radar portable nommé #n.",
+	"messages.securitycraft:portableRadar.withoutName": "#p est près de votre radar portable à #l.",
+	"messages.securitycraft:casseur de codeDisabled": "Le décrypteur a été désactivé via le fichier de configuration.",
```

Comment:
You accidentally changed `codebreaker` to `casseur de code` in the language key here. It should be `messages.securitycraft:codebreakerDisabled`.

Reply:
Still needs to be changed.